Jenkinsfile

```diff
@@ -131,7 +131,7 @@ try {
               sh "./scripts/test/hyriseServer_test.py clang-release"
               sh "./scripts/test/hyriseBenchmarkJoinOrder_test.py clang-release"
               sh "./scripts/test/hyriseBenchmarkFileBased_test.py clang-release"
-              sh "./scripts/test/hyriseBenchmarkTPCC_test.py clang-release"
+              sh "cd clang-release && ../scripts/test/hyriseBenchmarkTPCC_test.py ." // Own folder to isolate visualization
```

Comment:
to isolate export tests?

Reply:
This time I can't even blame on the time of day. :(